.travis.yml

```diff
@@ -6,7 +6,7 @@ addons:
     - python-scipy
 install:
   - pip install flake8  # Not yet available in Ubuntu precise (https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise)
-  - pip install -r requirements.txt
```

Comment:
What do we use `requirements.txt` for ? I don't get what is its role, isn't `setup.py` in charge of managing dependencies ?